client/src/pages/AdminDashboard.tsx

```
@@ -1,307 +1,201 @@
+        eventClick={handleEventClick}
+      />
+      <p className="mt-2 text-sm text-muted-foreground">
+        Cliquez sur un rendez-vous annulé pour le supprimer.
+      </p>
+    </div>
+  );
+}
+tsx
+Copier le code
+// AvailabilityManagement.tsx
+import React, { useState, useEffect } from 'react';
```

Biome (2.1.2)

[error] Shouldn't redeclare 'React'. Consider to delete it or rename it.

'React' is defined here:


(lint/suspicious/noRedeclare)